site/content/faq/somethings-not-working-what-should-i-do.md

```diff
@@ -54,17 +54,13 @@ you.
 
 Be aware that lots of webapps do weird and wonderful things, often in bizarre ways. We can't second guess them, so when asking for help you'll need to give us sanitized examples to help us figure out what's going on.
 
-###  Ask on IRC
+###  Ask on Slack
 
-Many of ZAP contributors hang out on the Libera Chat [#zaproxy](https://web.libera.chat/#zaproxy) channel which you can access via that link or your favourite irc client.
-
-###  Ask on Twitter
-
-You can ask [@psiinon](https://twitter.com/psiinon) short questions directly
-on twitter, although whether he's online will be timezone and holiday
-dependent.
+You can chat with the ZAP developers on [Slack](https://www.zaproxy.org/slack/), if you have not already joined then you will need an [invite](https://www.zaproxy.org/slack/invite).
 
 ###  Raise an Issue
 
 You can raise issues (bugs or new features) at the [issue
 tracker](https://github.com/zaproxy/zaproxy/issues/new).
```

Comment:
Could you tweak this to use: https://github.com/zaproxy/zaproxy/issues/new/choose

That way people won't get the blank template

Reply:
Or leave it, and I'll catch it in another PR.
There seem to be a bunch we should check/change: https://github.com/search?q=repo%3Azaproxy%2Fzaproxy-website%20%22https%3A%2F%2Fgithub.com%2Fzaproxy%2Fzaproxy%2Fissues%2Fnew%22&type=code